library/core/src/ptr/alignment.rs

```diff
@@ -1,9 +1,6 @@
-<<<<<<< HEAD
-#[cfg(not(feature = "ferrocene_certified"))]
-=======
 #![allow(clippy::enum_clike_unportable_variant)]
 
->>>>>>> pull-upstream-temp--do-not-use-for-real-code
+#[cfg(not(feature = "ferrocene_certified"))]
```

Comment:
This may have been mistakenly added. `NonZero` does not have it on the other side?

Reply:
hm, it's not in the certified subset (if that's what you are asking)